soql-server-pg/src/main/scala/com/socrata/pg/server/QueryServer.scala

```diff
@@ -33,7 +33,7 @@ import com.socrata.http.server.routing.SimpleResource
 import com.socrata.http.server.routing.SimpleRouteContext._
 import com.socrata.http.server.util.Precondition._
 import com.socrata.http.server.util.RequestId.ReqIdHeader
-import com.socrata.http.server.util.handlers.{NewLoggingHandler, ThreadRenamingHandler}
+import com.socrata.http.server.util.handlers.{LoggingOptions, NewLoggingHandler, ThreadRenamingHandler}
```

Comment:
Can the import be removed as well?

Reply:
NewLoggingHandler is still used.  We just configure it not to print its name.

ThreadRenamingHandler(NewLoggingHandler(logOptions)